src/common/user-store/user-store.ts

```diff
@@ -91,10 +91,6 @@ export class UserStore extends BaseStore<UserStoreModel> /* implements UserStore
    */
   @observable syncKubeconfigEntries!: StoreType<typeof DESCRIPTORS["syncKubeconfigEntries"]>;
 
-  @computed get resolvedShell(): string | undefined {
-    return this.shell || process.env.SHELL || process.env.PTYSHELL;
```

Comment:
By removing this does it mean we no longer fall back on process.env values? Not necessary anymore?

Reply:
It is not necessary anymore since user information reads from the system